newIDE/app/src/stories/componentStories/StorageProviders/SaveToStorageProviderDialog.stories.js

```diff
@@ -24,7 +24,7 @@ export default {
 
 export const UserNotAuthenticated = () => (
   <AuthenticatedUserContext.Provider
-    value={fakeNotAuthenticatedAuthenticatedUser}
+    value={fakeAuthenticatedUserNotAuthenticated}
```

Comment:
This naming is a bit misleading 😅 but I'm not sure I have better

Reply:
or `fakeNotAuthenticatedUser` since you used `fakeSilverAuthenticatedUser`

Reply:
true haha